tests/views/test_path.py

```diff
@@ -14,12 +14,15 @@ def test_invalid_repo_name(self):
         self.assertEqual(parsed_response_data['errors'][0]['message'], 'Not Found')
 
     def test_valid_repo_name_for_manifest(self):
-        headers = {'Accept': 'application/vnd.docker.distribution.manifest.v2+json'}
-        response = self.test_client.get('/v2/redhat/foo/manifests/1.25.1-musl', headers=headers)
+        # #3303: verify multi-valued headers too
+        # manifest lists are evaluated first, so pass a longer media type that
```

Comment:
can you elaborate more this please?

Reply:
The code first checks if manifest_list_mediatype is in accept_headers. If the client sent a string that *includes* 'application/vnd.docker.distribution.manifest.list.v2+json like in my +junk test, as well as a valid manifest v2 media type, then you'd get a false match for a manifest list.